### PR TITLE
Completed 14.10.2022 tasks

### PR DESCRIPTION
**Pull request by Wegrzynk**

### CODE REVIEW

File: Kacper Węgrzynowski/MiddleofLinkedList.cs [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+            counter++;
+        }
+        counter = counter / 2 + 1;
+        for (; counter>1; counter--){
+            head.val = head.next.val;
+            head.next = head.next.next;
+        }
+        return head;
+    }
+}
```
Comment (ArsenEloglian) [inline Kacper Węgrzynowski/MiddleofLinkedList.cs:32]: please add endline here

File: AttendanceList.txt [hunk 2]
```diff
@@ -5,4 +5,9 @@
     14.10.2022
     ....
   }
-*/
+  Kacper Węgrzynowski
+  {
+    07.10.2022
+    14.10.2022
+  }
+*/
```
Comment (ArsenEloglian) [inline AttendanceList.txt:13]: please add your changes below of */

**Review by ArsenEloglian — CHANGES_REQUESTED** [review]

File: Kacper Węgrzynowski/MiddleofLinkedList.cs (hunk 1 above)
Comment (Wegrzynk) [reply Kacper Węgrzynowski/MiddleofLinkedList.cs]: Pushed a new commit fixing this issue

File: AttendanceList.txt (hunk 2 above)
Comment (Wegrzynk) [reply AttendanceList.txt]: Pushed a new commit fixing this issue

**Review by ArsenEloglian — APPROVED** [review]

**Review by markow1997 — APPROVED** [review]

**Comment by Wegrzynk** [comment]
@ArsenEloglian I have similar problem to @WashingtonD in that i cannot merge this PR, because 4 of the older commits are unsigned. git commit -s doesn't seem to work. Should i do a rebase to sign these older commits, or is there a simpler alternative?

**Comment by ArsenEloglian** [comment]
> @ArsenEloglian I have similar problem to @WashingtonD in that i cannot merge this PR, because 4 of the older commits are unsigned. git commit -s doesn't seem to work. Should i do a rebase to sign these older commits, or is there a simpler alternative?

My proposition is to close this PR, create a new branch from the "main" branch and commit using flag -s (please remember to configure git before starting work with git:
git config --global user.name "FIRST_NAME LAST_NAME"
git config --global user.email "MY_NAME@example.com"
)

**Pull request closed without merging**